test.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+echo " -------- Building test image -------------"
+docker build -t apluslms/grading-z3:test .
+
+echo " -------- Executing test image ------------"
+if docker run --rm -it -v `pwd`:/submission apluslms/grading-z3:test python3 -c "from z3 import *; s = Solver(); s.add(Int('x') < 2*Int('y')); s.check(); print(s.model());"; then
```

Comment:
Maybe this test is fine for the repo. Can you move the python part to `test.py` for example, like with minizinc repo.

Reply:
Done